package.json

```diff
@@ -1,23 +1,26 @@
 {
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "build": "node packages/react-scripts/scripts/build.js",
+    "build": "cd packages/react-scripts && node scripts/build.js",
```

Comment:
Why did this change in this PR?  Do workspaces have an issue with working directories?

Reply:
I don't remember. I think it might not be related to YW but to Yarn in general (which sets cwd to the project you run the script in).